config/view.go

```diff
@@ -6,7 +6,22 @@ import (
 	"github.com/foolin/goview"
 )
 
-func GoviewConfig() goview.Config {
+func AppGoviewConfig() goview.Config {
+	return goview.Config{
+		Root:      "templates",
+		Extension: ".html",
+		Master:    "layouts/application",
+		Partials:  []string{},
+		Funcs: template.FuncMap{
+			"sub": func(a, b int) int {
```

Comment:
While this is part of the example on the repository, we don't make use of this, right? So it Can be removed 😅

Reply:
Yes, agree! Fixed in 7311140562c495d8bd27936512318ad82fc1b637